Solutions/Windows Security Events/Hunting Queries/masquerading_files.yaml

```diff
@@ -1,6 +1,8 @@
 id: 60304ebf-ebdd-4869-a702-e0216d90ab46
 name: Masquerading files
 description: |
+  'Malware often mimics system process names like svchost.exe to hide. Analysts should query for svchost.exe, excluding known SIDs and legitimate paths.'
```

Comment:
Malware writers often use windows system process names like svchost.exe to hide malicious activities. Query searches for execution of process svchost.exe, filtering out execution by well-known SIDs and from legitimate paths.